code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -68,6 +68,9 @@ GLOBAL_VAR(bomb_set)
 		else
 			to_chat(user, "<span class='notice'>You need to deploy [src] first.</span>")
 		return
+	else if(istype(O, /obj/item/disk/plantgene))
+		to_chat(user, "<span class='warning'>You try to plant the disk, but despite rooting around, it won't fit! After you branch out to read the instructions, you find out where the problem stems from. You've been bamboo-zled, this isn't a nuclear disk at all!</span>")
```

Comment:
I want to absolutely cry at this pun, but I guess it is **pun**ishment for those who make this mistake. Not an objection, but good god this hurts me.

*I only realised that pun above was a thing as I was typing this out. Its 1AM and I need help*

Reply:
May have to visit the branch manager that suggested it